### DISCUSSION TITLE
Where is color settings for this part of window's border in i3's config?

### DISCUSSION THREAD

**Discussion by arash28134** [i3/i3#5978] [Q&A] [answered]
When I open a new window in a workspace with no other windows, there's a blue strip on window's border that is lighter than rest of the border:

![2024-04-03_20-04](https://github.com/i3/i3/assets/74412308/c57744d0-0c2f-42ab-bba2-eb49b6660bd0)

(it goes away when I open another window in the workspace and the whole border becomes single-colored)

I read through the documentation but still couldn't find what part of i3's config file changes the color of this strip.

**Comment by okraits** [comment, marked as answer]
https://i3wm.org/docs/userguide.html#client_colors

The right border in this case is painted in the indicator color.
